clients/web/src/stylesheets/widgets/browserWidget.styl

```diff
@@ -0,0 +1,4 @@
+.g-hierarchy-widget-container
+  max-height calc(100vh - 400px)
+  min-height 200px
```

Comment:
Not strictly.  I added it in case somebody has a very small screen, then the hierarchy widget will still appear.  It's not ideal, but there are many bigger problems small screens anyway.